src/lib/Actor.hs

```diff
@@ -32,7 +32,7 @@ type Actor a = Chan a -> IO ()
 newtype PChan a = PChan { sendPChan :: a -> IO () }
 
 sendOnly :: Chan a -> PChan a
-sendOnly chan = PChan $ writeChan chan
+sendOnly chan = PChan $ \ !x -> writeChan chan x
```

Comment:
Is this shallow forcing sufficient? I imagine this is connected to our recent discussion about sending unevaluated thunks to other threads, right?

Reply:
I doubt it's really sufficient; I put it in there defensively.  In manual testing, it currently doesn't appear that evaluation work leaks into the driver thread, but this registers to me as far from bullet-proof.  Time to add `NFData` and use `deepSeq`?